src/options.cpp

```diff
@@ -2691,6 +2691,7 @@ std::string options_manager::show( bool ingame, const bool world_options_only )
             // keybinding screen erased the internal borders of main menu, restore it:
             draw_borders_internal( w_options_header, mapLines );
         } else if( action == "QUIT" ) {
+            catacurses::clear();
```

Comment:
Possibly it would be better to put `catacurses::clear()` before drawing certain screens instead of when exiting options screen as window would be cleared on "Save changes"/"Abort World generation " popups.

![image](https://user-images.githubusercontent.com/16213433/71171711-7fdb1580-226f-11ea-965e-f4a91c9cd950.png)

![image](https://user-images.githubusercontent.com/16213433/71171993-2e7f5600-2270-11ea-8648-af87cb4df118.png)